database/migrations/0000_init_helpers.sql

```diff
@@ -0,0 +1,36 @@
+-- 0000 migration
+
+-- Helpers
+
+-- In the previous version of the app, short_id was 7-14 characters long.
+-- To make migration to v2 easier, we will use a 15 character long short_id,
+-- so we can easily distinguish between v1 and v2 short_ids.
+CREATE DOMAIN short_id_v2 AS VARCHAR(15);
+
+CREATE DOMAIN varchar_short AS VARCHAR(50);
+CREATE DOMAIN varchar_normal AS VARCHAR(255);
+CREATE DOMAIN varchar_long AS VARCHAR(2500);
+
+CREATE DOMAIN varchar_email AS VARCHAR(255) CHECK (VALUE ~ '^.+@.+\..+$');
+CREATE DOMAIN varchar_fqdn AS VARCHAR(253);
+CREATE DOMAIN varchar_guid AS VARCHAR(36);
+CREATE DOMAIN varchar_password AS VARCHAR(36);
+CREATE DOMAIN varchar_slug AS VARCHAR(100);
+CREATE DOMAIN varchar_uri AS VARCHAR(2083);
```

Comment:
I mention in a previous comment about being consistent with data types. You are creating a domain here for url, using varchar. However, I know I saw at least one or two url columns using TEXT as the datatype. Again, neither options are incorrect-I think it would just bode well to try and stay as consistent as possible with your table structures and the data types you are using.

Reply:
I agree. At the moment I'm not sure which columns I missed with the `varchar_url`, but I do want to use that domain for all url columns.